### PR TITLE
Trial Project to review

### PR DESCRIPTION
**Pull request by JuanDBta**
In this branch you can find the following changes:
Add index.html
Add css folder with styles.css
Add images folder

### CODE REVIEW

File: css/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (JuanDBta) [inline css/index.html:10]: Hey Juan, great job doing this project! I would like to give you some suggestions, so you can make your code more readable and professional...here we go...
script tags usually go into the head tag, please review it...one reason is it ensures variables and functions are defined before they are used, and the second reason is that it allows the script to be downloaded and executed before the page content is rendered, resulting in a faster page load time.

File: css/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    <header id="header">
+      <div>This is a Landing Page</div>
```
Comment (JuanDBta) [inline css/index.html:13]: For your practice, headline goes better with "h1" tag...please think about it....using the "h1" tag for the page title is a best practice that can improve the accessibility, SEO, structure, and consistency of a web page.

File: css/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <div>This is a Landing Page</div> 
+      <div><img id="header-img" src="Images/IMG_5372.JPG" alt="Photo of a puppy"/ width="300px"></div>
+      
+      <div><nav id="nav-bar">
```
Comment (JuanDBta) [inline css/index.html:16]: Nice work Juan, it looks like you have put great effort and time into this project, so in order to give you a hand, kindly review this line, the"nav" tag is enough, you don´t need the "div" tag here...using both "nav" and "div" in the same line may create confusion and redundancy in your HTML code, which can make it more difficult to read and maintain...I hope this will be useful in your future coding journey.

File: css/index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        <ul>
+          <li><a class="nav-link" href="#header">Index</a></li>
+          <li><a class="nav-link" href="#video">Video</a></li>
+          <li><a class="nav-link" href="#header">Contact</a></li>
+        </ul>
+      </nav></div>
+
+    </header>
+    
+    <div>
```
Comment (JuanDBta) [inline css/index.html:26]: It is a better code practice to use semantic tags as "section" instead of "div"...sorry for being so detailed with you!  Using semantic tags like <section> can make the code more readable and easier to understand for other developers who may be working on the project. Semantic tags provide contextual clues about the purpose and structure of the content within them, which can make the code more self-explanatory and easier to navigate.

File: css/index.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+     <video id="video" src="video/Video de WhatsApp 2023-02-19 a las 18.56.38.mp4" controls alt="Video of a puppy" width="300px"/>
+    </div>
+
+    <div>
```
Comment (JuanDBta) [inline css/index.html:30]: again semantic tags are recommended...

File: css/index.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+      <form id="form" action="" method="post">
+        <input id="email" type="email" name="email" require placeholder="We need your email"></input>
+        <input id="submit" type="submit">Submit the email</input>
+      </form>
+    </div>
+
+    <script  src="js/index.js"></script>
```
Comment (JuanDBta) [inline css/index.html:37]: remember my first suggestion, please...You are pretty close to making code in a more professional way...good luck with the corrections...see you soon!

File: css/index.html (hunk 1 above)
Comment (jerryowusu) [reply css/index.html:10]: Kindly provide a reason as to why you are requesting each of the changes. This will help make it easy to understand and implement the changes. :+1:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @JuanDBta , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost there


## Highlights
Gave at least two comments in the review :heavy_check_mark: 
Used the github inline comments feature :heavy_check_mark: 

## Required Changes ♻️

- [ ] Kindly consider using the "Review changes" feature from Github since it is part of the project requierements and serve as summary for your review. Check [this link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) for more information on that.
- [ ] Kindly provide a reason as to why you are requesting each of the changes. This will help make it easy to understand and implement the changes. :+1: 


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/index.html (hunk 3 above)
Comment (AbelG101) [reply css/index.html:16]: - [ ] Hey Juan ✋, Please make sure to make your comments sound more polite and less of an order, and always make sure you provide a reason as to why you request changes. I like what you've done with the rest of your comments they're more polite and detailed in the changes you requested.
- [ ] The comment you made above doesn't sound that **polite** so kindly rephrase it to sound more gracious (avoid the use of exclamation marks as they indicate strong emotions) and also make sure you provide a **"Why"** for your requested change. ✔

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @JuanDBta 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the one you made._
- You're almost done you just have to make 1 change and you're on your way 🚀

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amon-cofie — APPROVED** [review]
Hi @JuanDBta  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 You have properly made the changes requested 👍 
🚀 You used polite words 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
